src/_components/shared/bump_cta.erb

```diff
@@ -0,0 +1,4 @@
+<bump-cta>
+  <p><strong>Bump.sh</strong>, an API doc platform for tech writers & engineers</p>
+  <a class="button" href="https://bump.sh">Get started for free <%= svg "images/icons/arrow-right.svg" %></a>
```

Comment:
```suggestion
  <a class="button" href="https://bump.sh/users/sign_in">Get started for free <%= svg "images/icons/arrow-right.svg" %></a>
```

I recommend that we directly get people to signup, hoping for a better conversion rate from visitor to user.